plugins/serializers/prometheusremotewrite/prometheusremotewrite.go

```diff
@@ -45,6 +46,26 @@ func (s *Serializer) SerializeBatch(metrics []telegraf.Metric) ([]byte, error) {
 		labels = s.appendCommonLabels(labels[:0], metric)
 		var metrickey MetricKey
 		var promts prompb.TimeSeries
+
+		if metric.Type() == telegraf.Histogram {
+			if ok, fh := tryGetNativeHistogram(metric); ok {
+				metrickey, promts = getPromNativeHistogramTS(metric.Name(), labels, *fh, metric.Time())
```

Comment:
I suggest merging `getPromNativeHistogramTS` and `tryGetNativeHistogram` and do
```suggestion
			if key, data := tryConvertToNativeHistogram(metric); key != "" && data != nil {
```

Reply:
sounds good! that does simplifies things